recipes/pylint-plugin-utils/meta.yaml

```diff
@@ -0,0 +1,42 @@
+{% set name = "pylint-plugin-utils" %}
+{% set version = "0.2.4" %}
+{% set sha256 = "6844b4d0525aa41ff69c5d44c6edc640e3dab98446abeb499501757f09a3cd71" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  {% set pypi_name = name.replace("_", "-") %}
+  fn: {{ name }}-{{ version }}.tar.gz
+  url: https://pypi.io/packages/source/{{ pypi_name[0] }}/{{ pypi_name }}/{{ pypi_name }}-{{ version }}.tar.gz
+  sha256: {{ sha256 }}
+
+build:
+  number: 0
+  script: python setup.py install --single-version-externally-managed --record record.txt
+
+requirements:
+  build:
+    - python
+    - setuptools
+    - pylint
+    - astroid
+
+  run:
+    - python
+    - pylint
+    - astroid
+
+test:
+  imports:
+    - pylint_plugin_utils
+
+about:
+  home: https://github.com/landscapeio/pylint-plugin-utils
+  license: GPL 2
```

Comment:
Proposing license file be included in PR ( https://github.com/landscapeio/pylint-plugin-utils/pull/4 ).